src/views/Predictions/hooks/useOnNextRound.ts

```diff
@@ -15,11 +21,14 @@ const useOnNextRound = () => {
     if (swiper && currentEpoch !== undefined && previousEpoch !== undefined && currentEpoch !== previousEpoch) {
       const currentEpochIndex = rounds.findIndex((round) => round.epoch === currentEpoch)
 
+      // Fetch data on current unclaimed bets
+      dispatch(fetchCurrentBets({ account, roundIds: rounds.map((round) => round.id) }))
+
```

Comment:
When the round changes we go and get the bets displayed again to keep it in sync.